packages/hemera-mongo-store/index.js

```diff
@@ -4,10 +4,15 @@ const Mongodb = require('mongodb')
 const ObjectID = Mongodb.ObjectID
 const MongoStore = require('./store')
 const StorePattern = require('hemera-store/pattern')
+const serialize = require('mongodb-extended-json').serialize
+const deserialize = require('mongodb-extended-json').deserialize
 
 exports.plugin = function hemeraMongoStore (options, next) {
   const hemera = this
   const topic = 'mongo-store'
+  const serializeResult = (result) => {
+    return (options.serializeResult === true) ? serialize(result) : result
```

Comment:
I consolidated it into one function so I didn't have to repeat myself calling the conditional 6+ times. :) But I can change it if need be.